temporal-sdk/src/test/java/io/temporal/activity/DefaultActivityOptionsSetOnWorkflowTest.java

```diff
@@ -68,6 +92,18 @@
           .setActivityImplementations(new TestActivityImpl())
           .build();
 
+  @Rule
```

Comment:
Do we need a separate rule, or can we re-use the existing rule above?

Reply:
Ah good point, we can re-use! Thanks!